src/api/common/schemas/balance-sheet-options.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "balance-sheet-options",
+  "description": "Options for getBalanceSheet",
+  "type": "object",
+  "properties": {
+    "hotwallet": {
```

Comment:
I think it would be simpler to just always make it an array and make the name more generic and descriptive like "excludedAddresses".